docs/documentation/head/connect.md

```diff
@@ -300,38 +300,42 @@ Connection conn = DriverManager.getConnection(url);
 
 * **defaultRowFetchSize** = int
 
-	Determine the number of rows fetched in `ResultSet`
-	by one fetch with trip to the database. Limiting the number of rows are fetch with 
-	each trip to the database allow avoids unnecessary memory consumption 
-	and as a consequence `OutOfMemoryError`.
+    Determine the number of rows fetched in `ResultSet`
+    by one fetch with trip to the database. Limiting the number of rows are fetch with 
+    each trip to the database allow avoids unnecessary memory consumption 
+    and as a consequence `OutOfMemoryError`.
 
-	The default is zero, meaning that in `ResultSet` will be fetch all rows at once. 
-	Negative number is not available.
+    The default is zero, meaning that in `ResultSet` will be fetch all rows at once. 
+    Negative number is not available.
 
 * **loginTimeout** = int
 
-	Specify how long to wait for establishment of a database connection. The
-	timeout is specified in seconds. 
+    Specify how long to wait for establishment of a database connection. The
+    timeout is specified in seconds. 
 
 * **connectTimeout** = int
 
-	The timeout value used for socket connect operations. If connecting to the server
-	takes longer than this value, the connection is broken. 
-	The timeout is specified in seconds and a value of zero means that it 	is disabled.
+    The timeout value used for socket connect operations. If connecting to the server
+    takes longer than this value, the connection is broken. 
+    The timeout is specified in seconds and a value of zero means that it 	is disabled.
 
 * **socketTimeout** = int
 
-	The timeout value used for socket read operations. If reading from the
-	server takes longer than this value, the connection is closed. This can
-	be used as both a brute force global query timeout and a method of
-	detecting network problems. The timeout is specified in seconds and a
-	value of zero means that it is disabled.
+    The timeout value used for socket read operations. If reading from the
+    server takes longer than this value, the connection is closed. This can
+    be used as both a brute force global query timeout and a method of
+    detecting network problems. The timeout is specified in seconds and a
+    value of zero means that it is disabled.
+
+* **sslResponseTikmeout** = int
```

Comment:
s/Tikmeout/Timeout/

Reply:
There's also bunch of unrelated whitespace conversion of tabs/spaces to this and README. Guessing it's from an editor autosave.

Reply:
ugh...